myuw/static/js/card/reg_status.js

```diff
@@ -165,6 +166,30 @@ var RegStatusCard = {
             return;
         }
 
+        // Retrieve pending majors and minors for this quarter, if they exist
+        var profile = WSData.profile_data();
+
+        var pending_minors = [];
+        var pending_majors = [];
+
+        for(i = 0; i < profile.term_majors.length; i++){
```

Comment:
can you turn these for loops into a method?  pending_majors = has_pending(profile.term_majors) maybe?